container-images/scripts/build_llama_and_whisper.sh

```diff
@@ -106,8 +106,9 @@ dnf_install_mesa() {
 }
 
 dnf_install_epel() {
+  local rpm_exclude_list="selinux-policy,container-selinux"
   local url="https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm"
-  dnf reinstall -y "$url" || dnf install -y "$url"
+  dnf reinstall -y "$url" || dnf install -y "$url" --exclude "${rpm_exclude_list}"
```

Comment:
If this was me I'd create a dnf_install function (and rename the old one) and only call that to ensure --exclude is passed every single time to dnf install, to ensure we never install these by mistake...

Also, can the "dnf reinstall" install it by mistake?

Reply:
I have no idea on the reinstall and why that was added.  I will merge this and then we can revisit by creating a dnf install bash command that does what you suggest.